packages/ember-runtime/lib/mixins/enumerable.js

```diff
@@ -1118,4 +1127,63 @@ if (isEnabled('ember-runtime-computed-uniq-by')) {
   });
 }
 
+if (isEnabled('ember-runtime-enumerable-includes')) {
+  Enumerable.reopen({
+    /**
+      Returns `true` if the passed object can be found in the enumerable.
+      ```javascript
+      [1, 2, 3].includes(2);                     // true
+      [1, 2, 3].includes(4);                     // false
+      [1, 2, undefined].includes(undefined);     // true
+      [1, 2, null].includes(null);               // true
+      [1, 2, NaN].includes(NaN);                 // true
+      ```
+      @method includes
+      @param {Object} obj The object to search for.
+      @return {Boolean} `true` if object is found in the enumerable.
+      @public
+    */
+    includes(obj) {
+      assert('Enumerable#includes cannot accept a second argument "startAt" as enumerable items are unordered.', arguments.length === 1);
+
+      var len = get(this, 'length');
+      var idx, next;
+      var last = null;
+      var found = false;
+
+      var context = popCtx();
+
+      for (idx = 0; idx < len && !found; idx++) {
+        next = this.nextObject(idx, last, context);
+
+        found = obj === next || (obj !== obj && next !== next);
+
+        last = next;
+      }
+
+      next = last = null;
+      context = pushCtx(context);
+
+      return found;
+    },
+
+    without(value) {
+      if (!this.includes(value)) {
```

Comment:
👍  to updating the docs and keeping the current behavior. Great catch.

The `without` method on enumerables and on mixins just should just 🔥 🔥 🔥. On enumerables we can just use `reject` which seems more sane, and on mixins the functionality is only used for native array prototype application. I attempted killing it once and ran out of steam.